app/views/support_interface/validation_errors/vendor_api/index.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for :title, 'Vendor API validation errors' %>
+
+<% content_for :before_content do %>
+  <%= breadcrumbs({
+    'Performance': support_interface_performance_path,
+    'Validation errors': support_interface_validation_errors_path,
+    'Vendor API': nil,
+  }) %>
+<% end %>
+
+<p class="govuk-body"><%= govuk_link_to 'Validation error summary', support_interface_validation_errors_vendor_api_summary_path %></p>
+
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header govuk-!-width-three-quarters">Attribute error</th>
+      <th class="govuk-table__header govuk-table__header--numeric govuk-!-width-one-quarter">Error count</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+  <% @grouped_column_error_counts.each do |(request_path, error, message), count| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          <span class="govuk-!-font-weight-regular"><%= govuk_link_to request_path, support_interface_validation_errors_vendor_api_search_path(request_path: request_path) %>:</span>
```

Comment:
(As discussed, probably one for a future iteration)

Grouping by exact request path might not be as useful as grouping by request path _up to ids_. It may be nicer to have them grouped by e.g. `/application/{id}/offer` (and capture the id in details somewhere), so we can see that the offer endpoint gets lots of validation errors, rather than by `/application/420/offer`, where we would have to do some maths to work out how many offers resulted in a validation error.